setup.cfg

```diff
@@ -94,7 +94,7 @@ sentry =
 # Currently, the black formatter doesn't wrap long strings: https://github.com/psf/black/issues/182#issuecomment-385325274
 # We already have a lot of E501's - these are lines black didn't wrap.
 # But rather than append # noqa: E501 to all of them, we just ignore E501 for now.
-extend-ignore = E203,E501,E402,E731,B007,B009,B010,B011,B020,B023,B024,B026,B027
```

Comment:
I'm disabling the new lint rule here and will follow up to re-enable it -- just in case a few sneak in as implicit conflicts